services/datamanager/builtin/builtin.go

```diff
@@ -550,10 +550,8 @@ func (svc *builtIn) Update(ctx context.Context, cfg *config.Config) error {
 			}
 		} else if attributes.Disabled {
 			// if disabled, make sure that it is closed, so it doesn't keep collecting data.
-			collector, md, err := svc.getCollectorFromConfig(attributes)
-			if err != nil {
-				svc.logger.Errorw("collector ", attributes.Name, " was not found", "info", err)
-			} else {
+			collector, md, ok := svc.getCollectorFromConfig(attributes)
```

Comment:
Do you remember why this was previously considered an error? Asking just to double check that this isn't an indication that something is wrong (looking at the code it doesn't look like it, but making sure)

Reply:
Looks like it was added newly [here](https://github.com/viamrobotics/rdk/pull/1510/files#diff-ffb86a3e2b364cc470f33937bb1ffb08ca2ef9d01b2734320daea7bee24a8943), so that was just something I missed in the review. Double checking with @alexis-wei here otherwise: a component/method gets removed from the map once it's disabled, and it's expected it won't be in the map if a config update doesn't include re-enabling that component/method.

Reply:
I think the new changes are good, I don't think the previous log needed to be an error, just a info warning to let the user know it wasn't found